CHANGELOG.md

```diff
@@ -297,7 +303,3 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [0.4.0]: https://github.com/cartesi/rollups/releases/tag/v0.4.0
 [0.3.0]: https://github.com/cartesi/rollups/releases/tag/v0.3.0
 [0.2.0]: https://github.com/cartesi/rollups/releases/tag/v0.2.0
-[0.1.3]: https://github.com/cartesi/rollups/releases/tag/v0.1.3
-[0.1.2]: https://github.com/cartesi/rollups/releases/tag/v0.1.2
-[0.1.1]: https://github.com/cartesi/rollups/releases/tag/v0.1.1
-[0.1.0]: https://github.com/cartesi/rollups/releases/tag/v0.1.0
```

Comment:
Any reason for deleting the `Unreleased` and `pre-0.2.0` releases from the list?

Reply:
No reason, I'll put those back.

Reply:
Actually, my VSCode is removing those when I save the file.

Reply:
That was the reason: https://github.com/DavidAnson/markdownlint/blob/v0.31.1/doc/md053.md